### ISSUE TITLE
Syntax highlighting and comments broken

### ISSUE THREAD

**Issue by brupelo** [babel/babel-sublime#389]
**Version: v8.6.3**

Consider this simple snippet:

```
export default function Foo(props) {
  return (
    <div>
      <AppBar>
        <Toolbar>
          {user.role && (
            <>
              {user.role !== "Guest" && (
                <>
                </>
              )}
              {user.role === "Admin" && (
              )}
            <Link to="/logout" className={classes.link}>
              <Button>
              </Button>
            </Link>
            </>
          )}
        </Toolbar>
      </AppBar>
    </div>
  );
}
```

And now take a look to the comparison between the output provided by babel-sublime vs builting js VsCode's highlighter:

![showcase](https://bit.ly/3aRkh4L)

While VsCode highlight is perfect in Sublime using this plugin appears completely broken.

Also... in addition... when comment blocks of code in vscode is done properly, ie:

![showcase](https://bit.ly/3leST5t)

while with babel-sublime package fails in some cases.

**Comment by jorgebzf** [comment]
Estou tendo problemas com o Babel instalado no Sublime, a síntaxe fica quebrada!

**Comment by Thom1729** [comment]
Fixed in v10.0.0.
